### DISCUSSION TITLE
Help! Images and CSS not appearing

### DISCUSSION THREAD

**Discussion by divinefire53** [community/community#124884] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

https://divinefire53.github.io/lovelifepublishing1/

Hi all, I'm trying to post a school project online. I've got all my files in the main branch (3 html files, 1 css, and some images), and the html shows up but the css and images won't. 

In my "pages" section it's set to deploy from a branch, and says "Your GitHub Pages site is currently being built from the main branch."

I've taken out the reference to other folders in the code, so I have 
link rel="stylesheet" href="styles.css"
and 
img src="checkmarklistbluecircle.png" alt="List with checkmark" class="round" (instead of img src="images/checkmarklistbluecircle.png")

I have everything in lowercase, so I'm not trying to link to Checkmarklistbluecircle, for example. 

What am I doing wrong??

**Comment by CopperEagle** [comment]
Hi @divinefire53. Looks all working to me

![screenshot image](https://github.com/community/community/assets/137458753/63bd9ea0-b4e5-4831-8757-fd35fb998d45)

Looks cool :+1:

**Reply by divinefire53** [reply]
Thank you! It was one of those weird times where I've worked on it for like 4 hours over a few days and everything was the same, then I refreshed it like I had done a million times and everything magically appeared. What a relief (with just a few hours to the deadline)!
